BDArmory/Weapons/Missiles/MultiMissileLauncher.cs

```diff
@@ -333,6 +333,22 @@ void UpdateFields(MissileLauncher MLConfig, bool configurableSettings)
                 missileLauncher.shortName = MLConfig.shortName;
                 missileLauncher.blastRadius = -1;
                 missileLauncher.blastRadius = MLConfig.blastRadius;
+                missileLauncher.LoftMaxAltitude = MLConfig.LoftMaxAltitude;
+                missileLauncher.LoftRangeOverride = MLConfig.LoftRangeOverride;
+                missileLauncher.LoftAltitudeAdvMax = MLConfig.LoftAltitudeAdvMax;
+                missileLauncher.LoftMinAltitude = MLConfig.LoftMinAltitude;
+                missileLauncher.LoftAngle = MLConfig.LoftAngle;
+                missileLauncher.LoftTermAngle = MLConfig.LoftTermAngle;
+                missileLauncher.LoftRangeFac = MLConfig.LoftRangeFac;
+                missileLauncher.LoftVelComp = MLConfig.LoftVelComp;
+                missileLauncher.LoftVertVelComp = MLConfig.LoftVertVelComp;
+                //missileLauncher.LoftAltComp = LoftAltComp;
+                missileLauncher.LoftTermRange = MLConfig.LoftTermRange;
+                missileLauncher.LoftGuidanceModeTerminal = MLConfig.LoftGuidanceModeTerminal;
+                missileLauncher.pronavGain = MLConfig.pronavGain;
+                missileLauncher.loftState = 0;
+                missileLauncher.TimeToImpact = float.PositiveInfinity;
+                missileLauncher.initMaxAoA = MLConfig.maxAoA;
```

Comment:
if any of these settings are settings that are baked into the .cfg and/or internal values/not adjustable by the player in the SPH, they should go in the top section outside the if (configurableSettings) block to ensure they get properly passed along to the missile when updating fields.

Reply:
Right, good call.